packages/app/components/MemberCard/RoundedIcon.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Image, View } from "react-native";
+
+const RoundedIcon = ({ image }) => {
+    const roundedIconStyle = {
+        width: 60,
+        height: 60,
+        borderRadius: 30,
+        justifyContent: "center",
+        alignItems: "center",
+        overflow: "hidden",
+    };
+
+const imageStyle = {
+    aspectRatio: 1,
+    width: "100%",
+    height: "100%",
+    borderRadius: 15,
+};
+
+return (
+    <View style={roundedIconStyle}>
+        <Image
+        style={imageStyle}
+        source={image ? { uri: image } : require("@app/assets/no-pfp-icon.png")}
+        />
+    </View>
```

Comment:
Could you add click functionality to this icon? It should be a prop, so we can change the functionality from anywhere.